composer.json

```diff
@@ -27,6 +27,7 @@
         "symfony/class-loader": "~2.3",
         "symfony/dom-crawler": "~2.3",
         "symfony/css-selector": "~2.3",
+        "phpunit/phpunit": "4.5.*",
```

Comment:
PHPUnit 4 is not fully compatible with PHP 7 (but you have to keep it for PHP versions before 5.6).